package.json

```diff
@@ -14,7 +14,8 @@
     "usertool": "node ./build/tools/userClientTools.js",
     "directoryfix": "node ./build/tools/addRoomsToDirectory.js",
     "ghostfix": "node ./build/tools/ghostfix.js",
-    "chanfix": "node ./build/tools/chanfix.js"
+    "chanfix": "node ./build/tools/chanfix.js",
+    "moveRoomStore": "node ./build/tools/moveRoomStoreToDb.js"
```

Comment:
Is this tool needed if we have stuff in the migraiton?

Reply:
No, and I deleted it. I should also remove it from package.json